### ISSUE TITLE
Color for holocat background in prod is different from figma

### ISSUE THREAD

**Issue by ecurrencyhodler** [saving-satoshi/saving-satoshi#288]
**Describe the bug**
The background color for https://savingsatoshi.com/en/chapters/chapter-1/intro-2 is different from figma. I guess this is a question for Christoph first. 
<img width="1414" alt="Screenshot 2023-03-12 at 8 02 17 PM" src="https://user-images.githubusercontent.com/32662508/224598636-fa304e78-b7b5-4727-9c11-8066f244e026.png">

**Comment by vinayaksh42** [comment]
I think during one of the calls or under one of the issues it was decided to keep the background as it is right now for the demo and then update it for V1. (I am not sure tho, we should confirm it with @GBKS )

**Comment by GBKS** [comment]
Yes, we decided to not worry about this for the demo. As you can see in the [design system](https://www.figma.com/file/LqjK3Tpvd9KJ4buFArCJBQ/Saving-Satoshi?node-id=2019%3A22942&t=rkpLiQrj1RkPvUk0-1), one of the ideas (since very early on) was to theme chapters with different colors. Having worked more on the images now, I think a good approach is that we consider both the time of day and the story context for this. For example, in chapter 1, our hero is having breakfast and embarking on an exciting adventure. That provides us imagery set with morning sunlight and a generally happy/positive vibe. The 51% attack could be set in the evening and there's a battle happening, so maybe we go with moonlight and more aggressive red tones. The generated images then have a consistent color them, which the backgrounds would be based on (picking some of the darker tones for a simple 2-step gradient). In the demo specs, I experimented with some page-by-page adjustments (still similar tones, but slight differences), which make each page look a bit nicer, but slightly reduces consistency across page chapters. I think we definitely want the chapter-based color coding. Whether to make page-based adjustments is something we can figure out post-demo. So that's my thinking so far, happy to throw this out to the group for input.

<img width="876" alt="image" src="https://user-images.githubusercontent.com/695901/224636828-8782d852-2448-4ece-9e05-5e5ab9d57ced.png">

**Comment by afterburn** [comment]
As a note for whoever is going to implement this in the future:

I think it makes sense to be able to specify this color (or color gradient) via the lesson metadata, this will allow us to easily modify colors from a centralized location as opposed to implementing this on a per-lesson basis.

@GBKS I like the idea of color-coding lesson/chapters, helps set the mood a bit more.

**Comment by GBKS** [comment]
One of my favorite stories about whether people notice differences or not (about [this song](https://www.youtube.com/watch?v=zhl-Cs1-sG4)). I think there are a lot of things that people may not consciously notice, but contribute to the overall feel and quality of a product. 

<img width="753" alt="image" src="https://user-images.githubusercontent.com/695901/224937779-346dce78-e962-45b3-a950-ab6993ffe048.png">
